src/applications/vaos/appointment-list/components/AppointmentsPage/ApointmentsPageV2.jsx

```diff
@@ -0,0 +1,284 @@
+} from '../../../redux/selectors';
+import { GA_PREFIX, FETCH_STATUS } from '../../../utils/constants';
+import { scrollAndFocus } from '../../../utils/scrollAndFocus';
+import TabNav from './TabNav';
+import RequestExpressCare from './RequestExpressCare';
+import FutureAppointmentsList from '../FutureAppointmentsList';
+import PastAppointmentsList, {
+  getPastAppointmentDateRangeOptions,
+} from '../PastAppointmentsList';
+import ExpressCareList from '../ExpressCareList';
+import PageLayout from './PageLayout';
+import DowntimeNotification, {
+  externalServices,
+} from 'platform/monitoring/DowntimeNotification';
+import WarningNotification from '../../../components/WarningNotification';
+import ScheduleNewProjectCheetah from './ScheduleNewProjectCheetah';
+import Select from './Select';
+
+const pageTitle = 'VA appointments';
+
+const options = [
+  { label: 'Upcoming', value: 'upcoming' },
+  { label: 'Requested', value: 'requested' },
+  { label: 'Past', value: 'past' },
+  { label: 'Cancelled', value: 'cancelled' },
+];
+
+function AppointmentsPageV2({
+  cancelInfo,
+  closeCancelAppointment,
+  confirmCancelAppointment,
+  expressCare,
+  fetchFutureAppointments,
+  fetchExpressCareWindows,
+  fetchPastAppointments,
+  futureStatus,
+  isCernerOnlyPatient,
+  isWelcomeModalDismissed,
+  pendingStatus,
+  showCommunityCare,
+  showDirectScheduling,
+  showExpressCare,
+  showPastAppointments,
+  showScheduleButton,
+  showCheetahScheduleButton,
+  startNewAppointmentFlow,
+  startNewExpressCareFlow,
+  showHomePageRefresh,
+  updateAppointmentListFilter,
+}) {
+  useEffect(() => {
+    document.title = `${pageTitle} | Veterans Affairs`;
+
+    if (showExpressCare && futureStatus === FETCH_STATUS.notStarted) {
+      fetchFutureAppointments();
+    }
+
+    if (
+      expressCare.enabled &&
+      expressCare.windowsStatus === FETCH_STATUS.notStarted
+    ) {
+      fetchExpressCareWindows();
+    }
+  }, []);
+
+  useEffect(
+    () => {
+      if (isWelcomeModalDismissed) {
+        scrollAndFocus();
+      }
+    },
+    [isWelcomeModalDismissed],
+  );
+
+  useEffect(
+    () => {
+      if (
+        !cancelInfo.showCancelModal &&
+        cancelInfo.cancelAppointmentStatus === FETCH_STATUS.succeeded
+      ) {
+        scrollAndFocus();
+      }
+    },
+    [cancelInfo.showCancelModal, cancelInfo.cancelAppointmentStatus],
+  );
+
+  const isLoading =
+    pendingStatus === FETCH_STATUS.loading ||
+    expressCare.windowsStatus === FETCH_STATUS.loading ||
+    pendingStatus === FETCH_STATUS.notStarted ||
+    expressCare.windowsStatus === FETCH_STATUS.notStarted;
+
+  // TODO - Rename to 'AppointmentList' since the list now displays all appointment types.
+  const routes = (
+    <Switch>
+      <Route component={PastAppointmentsList} path="/past" />
+      <Route component={ExpressCareList} path="/express-care" />
+      <Route path="/" component={FutureAppointmentsList} />
+    </Switch>
+  );
+
+  function onChange(e) {
```

Comment:
Rather than having the dropdown update a filter stored in Redux, I think we should have it route to different pages, like we do with tabs. Implementing it that way means that each option in the dropdown has a different url, so you can use the browser back/forward button, as well as directly link each of the different views of your appointments. Additionally, I think it will make the past appointments view easier by keeping the logic for making additional fetches to get different ranges of past appointments, and the additional dropdown, in their own component.

Reply:
Done.